src/index.rst

```diff
@@ -7,34 +7,43 @@
 Axom
 ****
 
-Axom is a project in WCI/WSC that is funded by ECP/ATDM.
-Its principal goal is to provide a collection of robust and flexible software 
-components that serve as building blocks for LLNL simulation tools. The 
-emphasis is on sharing core infrastructure software amongst applications 
-rather than having different codes develop and maintain similar capabilities.
-
-A key objective of Axom is to facilitate integration of novel, 
-forward-looking computer science capabilities into LLNL simulation codes. 
-Thus, a central function of Axom is to enable and simplify data exchange 
-between applications and tools that Axom provides. To meet these 
-objectives, developers of Axom components emphasize the following features 
-in software design and implementation:
-
-  * Flexibility to meet the needs of a diverse set of applications
-  * High-quality, with well designed APIs, good documentation, tested well, high performance, etc.
-  * Consistency in software engineering practices
-  * Integrability so that components work well together and are easily adopted by applications
-
-The main drivers of the Axom project are to:
-
-  *  Provide the CS infrastructure foundation of the ECP/ATDM multi-physics application at LLNL
-  *  Support current ASC and other production applications and as they continue to evolve
-  *  Provide capabilities for LLNL research codes, proxy apps, etc. that simplify technology
-     transfer from research efforts into production applications
-
-==========
-Components
-==========
+Axom is an open source project that provides robust and flexible software 
+components that serve as building blocks for high performance scientific 
+computing applications. A key goal of the project is to have different 
+application teams co-develop and share general core infrastructure software 
+across their projects instead of individually developing and maintaining 
+capabilities that are similar in functionality but are not easily shared.
+
+An important objective of Axom is to facilitate integration of novel, 
+forward-looking computer science capabilities into simulation codes. 
+A pillar of Axom design is to enable and simplify the exchange of 
+simulation data between applications and tools. Axom developers 
+emphasize the following principles in software design and implementation:
+
+  * Start design and implementation based on concrete application use cases and maintain flexibility to meet the needs of a diverse set of applications
+  * Develop high-quality software that has well-designed APIs, good documentation, solid testing, and is robust and high performance
```

Comment:
"and highly performant"?

Reply:
Uhhh. I hate the term "performant". I think "high performance" is fine, but I will change if there is a revolt.

Reply:
How about "and provides high performance"?

Reply:
sure. I'll make the change.

Reply:
@white238 I pushed a change to the entire statement. Please take a look.

Reply:
Looks great